nav2_bringup/bringup/launch/nav2_bringup_launch_experimental.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
```

Comment:
I'd put the launch file in the nav2_experimental package as well. It makes sense to me to have a nav2_experimental/nav2_rl/nav2_turtlebot3_rl/launch directory. That should be added to the data_files section of setup.py so it gets installed as well.

Reply:
sure, I will make that change.